prefab_cloud_python/client.py

```diff
@@ -110,5 +123,9 @@ def post(self, path, body):
             "Authorization": f"Basic {auth_token}",
```

Comment:
nit: this can be simplified to `self.session.post(..., auth=("authuser", self.options.api_key))`